test/tests/selftest/pbs_expect.py

```diff
@@ -57,7 +57,7 @@ def test_attribute_case(self):
         # Set the Priority attribute on the queue but provide 'p' lowercase
         # Set other attributes normally
         a = {'enabled': 'True', 'started': 'True', 'priority': 150}
-        self.server.manager(MGR_CMD_SET, QUEUE, a, 'expressq', expect=True)
+        self.server.manager(MGR_CMD_SET, QUEUE, a, 'expressq')
```

Comment:
This test really doesn't make any sense without a call to expect(). Can you please add an explicit call to expect() after the call to manager() ?